libs/wingsdk/src/std/array.ts

```diff
@@ -125,6 +145,19 @@ export class Array {
     searchElement;
     throw new Error("Macro");
   }
+
+  /**
+   * Returns the array sorted without mutating the inital
+   *
+   * @macro ((arr, args) => { return args && args[0] ? [...(arr)].sort(args[0]) : [...(arr)].sort() })($self$, $args$)
+   *
+   * @param comparator Function used to determine the order of the elements.
+   * @returns same array, sorted
+   */
+  public sorted(comparator?: IComparator): Array {
```

Comment:
`sorted()` should also be available on MutArray, no?

Reply:
It's a good proposal; I implemented just sort() and sorted() according to what it was on the original spec but we can iterate it further

Reply:
Interesting; I'll put it on my todo list